Cabal/doc/installing-packages.rst

```diff
@@ -690,6 +698,22 @@ path options:
     ``$pkg``, ``$version``, ``$compiler``, ``$os``, ``$arch``, ``$abi``,
     ``$abitag``
 
+.. option:: --libsubdir=dir
+
+    For use with Setup.hs files build against a version of Cabal prior to 1.25.
+    For later versions of Cabal, this flag is basically deprecated, and you
+    should use ``--commonlibdir=dir``.
```

Comment:
With later versions of Cabal, you should prefer :option:\`--commonlibdir\` instead.